components/expand-collapse/README.md

```diff
@@ -1,6 +1,4 @@
-# Collapsible Containers
-
-## Expand Collapse Content [d2l-expand-collapse-content]
+# Expand Collapse Content [d2l-expand-collapse-content]
```

Comment:
Sure, you could add a quick summary there since that's what will be shown in the search results. We usually also add a [component demo](https://daylight.d2l.dev/components/card/) here too but it's probably not needed in this case since the page contains just a single component.